sdk/test/test_objectstore.py

```diff
@@ -7,14 +7,13 @@
 
 import unittest
 
-from basyx.objectstore import ObjectStore, ObjectProviderMultiplexer
+from basyx.object_store import ObjectStore, ObjectProviderMultiplexer  # type: ignore
```

Comment:
Could you add a comment above line 10 with the following schema: 

```Python
# (2024-09-12, somsonson)
# We tell mypy to ignore this type because of <reason>
```

That way, in the future, we don't have to guess why we do not care (or rather cannot test) type safety at this point.